registry/templates/donor/detail.html

```diff
@@ -76,6 +76,39 @@ <h2>Poznámky</h2>
         {% endwith %}
     </div>
 </div>
+
+<h2>Přepsání údajů</h2>
+{% with form=donors_override_form %}
+<form id="donorsOverrideForm" action="{{ url_for('donor.save_override') }}" method="POST">
+    {{ form.csrf_token }}
+    <table class="table table-striped table-hovered table-hover">
```

Comment:
Hover efekt bych u tabulky s jedním řádkem nedával.
```suggestion
    <table class="table table-striped table-hovered">
```

Reply:
Opraveno.